examples/fedpca_examples/dim_reduction/config.yaml

```diff
@@ -0,0 +1,12 @@
+# Parameters that describe server
+n_server_rounds: 3 # The number of rounds to run FL
+
+# Parameters that describe clients
+n_clients: 3 # The number of clients in the FL experiment
+local_epochs: 2 # The number of epochs to complete for client
+batch_size: 64 # The batch size for client training
+new_dimension: 12
+
+# checkpointing
```

Comment:
Maybe elaborate that these are the paths to the pre-computed principal components (`pca_path`) and the path where FL trained models will be saved during training (`checkpoint_path`)